src/tracing/writer.rs

```diff
@@ -65,6 +67,13 @@ impl<W: Write> TraceWriter<W> {
         self
     }
 
+    /// Sets the starting indentation level.
+    #[inline]
+    pub fn write_creation_codes(mut self, yes: bool) -> Self {
+        self.write_creation_codes = yes;
```

Comment:
I'd name it `code` instead of plural `codes`

Reply:
It follows the convention of `color_cheatcodes` because just like cheatcodes, creation codes can appear multiple times in a trace. If this reasoning doesn't convince you, I'll change it.